install_config/cluster_metrics.adoc

```diff
@@ -37,6 +37,10 @@ each node individually through the `/stats` endpoint. From there, Heapster
 scrapes the metrics for CPU, memory and network usage, then exports them into
 Hawkular Metrics.
 
+The storage volume metrics available on the kubelet are not available through
+the `/stats` endpoint, but are available through the ``/metrics` endpoint. See
```

Comment:
double backtick at the start of the /metrics bit?

Reply:
Fixed.